### DISCUSSION TITLE
Parcel v2 fails to build react app when using @import url() in scss module

### DISCUSSION THREAD

**Discussion by williamOnsare** [parcel-bundler/parcel#8488] [Q&A]
After [migrating](https://parceljs.org/getting-started/migration) from Parcel v1 to v2, building the same react app (CRA) using parcel keeps failing to resolve [Google fonts URL](https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600;700&display=swap) imported in stylesheet claiming its an **_unknown url scheme or pipeline 'https:'_**
 Kindly help.

The line causing failure is
```scss
@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600;700&display=swap');
```

which I also attempted without the _url()_ syntax like this
```css
@import "https://fonts.googleapis.com/css2?family=Quicksand:wght@300;400;500;600;700&display=swap";
```

 but still fails to build.

Here is a screenshot
![Parcel fails to resolve external url](https://user-images.githubusercontent.com/65705419/190975066-69e299b5-ab7b-4165-8478-4b91117e90b7.png)

Below is the `package.json` file

```json
{
  "name": "live-chat-widget",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "classnames": "^2.3.2",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-scripts": "5.0.1",
    "sass": "^1.54.9"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "build:widget": "parcel build src/index.js --no-source-maps --dist-dir docs"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
  "devDependencies": {
    "@parcel/transformer-sass": "2.7.0",
    "parcel": "^2.7.0"
  }
}
```

**Comment by iSplasher** [comment]
Just returned to a parcel project after a long time. Upgraded to 2.8.2 from 2.2.0 and this is the error I keep getting when I build. Seems ridiculous you can't important fonts from external URLs.
